<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainUI</class>
 <widget class="QMainWindow" name="MainUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Information</string>
  </property>
  <property name="windowIcon">
   <iconset theme="document-search"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_file">
       <attribute name="title">
        <string>File Information</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <property name="labelAlignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_file_icon">
          <property name="text">
           <string notr="true">icon</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="Line" name="line_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Owner:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_file_owner">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Group:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="label_file_group">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Permissions:</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0" colspan="2">
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Created:</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QLabel" name="label_file_created">
          <property name="toolTip">
           <string>Note: The time a file was created might be more recent than the time modified if the file permissions were changed recently.</string>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Last Modified:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Type:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>MimeType:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_file_mimetype">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_file_type">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="label_file_perms">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QLabel" name="label_file_modified">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>File Size:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_file_size">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_file_name">
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_deskedit">
       <attribute name="title">
        <string>XDG Shortcut</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="lblWorkingDir">
            <property name="text">
             <string>Working Dir:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QCheckBox" name="check_xdg_startupNotify">
            <property name="text">
             <string>Use startup notification</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="line_xdg_wdir">
            <property name="enabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="lblIcon">
            <property name="text">
             <string>Icon:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="lblCommand">
            <property name="text">
             <string>Command:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="lblComment">
            <property name="text">
             <string>Comment:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="line_xdg_command"/>
          </item>
          <item row="6" column="1">
           <widget class="QCheckBox" name="check_xdg_useTerminal">
            <property name="text">
             <string>Run in terminal</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="lblName">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0" alignment="Qt::AlignRight">
           <widget class="QLabel" name="lblOptions">
            <property name="text">
             <string>Options</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="line_xdg_name"/>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QLineEdit" name="line_xdg_comment"/>
          </item>
          <item row="2" column="2">
           <widget class="QToolButton" name="tool_xdg_getCommand">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>42</height>
             </size>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="icon">
             <iconset theme="edit-find">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QToolButton" name="tool_xdg_getDir">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>42</height>
             </size>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="icon">
             <iconset theme="edit-find">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_xdg_icon">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>64</width>
                <height>64</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>64</width>
                <height>64</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="text">
               <string notr="true"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="line_xdg_icon"/>
            </item>
           </layout>
          </item>
          <item row="4" column="2">
           <widget class="QToolButton" name="push_xdg_getIcon">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20000</width>
              <height>42</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>2</width>
              <height>2</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="icon">
             <iconset theme="edit-find">
              <normaloff>.</normaloff>.</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>39</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSave_As">
     <property name="title">
      <string>Save As</string>
     </property>
     <property name="icon">
      <iconset theme="document-save"/>
     </property>
     <addaction name="actionLocal_Shortcut"/>
     <addaction name="actionRegister_Shortcut"/>
    </widget>
    <addaction name="actionNew_Shortcut"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_File"/>
    <addaction name="actionOpen_Directory"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Shortcut"/>
    <addaction name="menuSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_File">
   <property name="icon">
    <iconset theme="document-open"/>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="window-close"/>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionSave_Shortcut">
   <property name="icon">
    <iconset theme="document-save"/>
   </property>
   <property name="text">
    <string>Save Shortcut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionLocal_Shortcut">
   <property name="icon">
    <iconset theme="document-new"/>
   </property>
   <property name="text">
    <string>Local Shortcut</string>
   </property>
  </action>
  <action name="actionRegister_Shortcut">
   <property name="icon">
    <iconset theme="bookmark-new"/>
   </property>
   <property name="text">
    <string>Register Shortcut</string>
   </property>
  </action>
  <action name="actionOpen_Directory">
   <property name="icon">
    <iconset theme="folder-open"/>
   </property>
   <property name="text">
    <string>Open Directory</string>
   </property>
  </action>
  <action name="actionNew_Shortcut">
   <property name="icon">
    <iconset theme="document-new"/>
   </property>
   <property name="text">
    <string>New Shortcut</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
